formicaio/docker-compose.yml

```
@@ -0,0 +1,29 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: formicaio_formicaio_proxy_1
+      APP_PORT: 52100
+    container_name: formicaio-app_proxy_1
+  formicaio_proxy:
```

Change name to proxy to make the readability better.
```suggestion
  proxy:
```